python/dgl/graphbolt/item_sampler.py

```diff
@@ -2,16 +2,53 @@
 
 from collections.abc import Mapping
 from functools import partial
-from typing import Iterator, Optional
+from typing import Callable, Iterator, Optional
 
 from torch.utils.data import default_collate
 from torchdata.datapipes.iter import IterableWrapper, IterDataPipe
 
+from ..base import dgl_warning
+
 from ..batch import batch as dgl_batch
 from ..heterograph import DGLGraph
 from .itemset import ItemSet, ItemSetDict
+from .minibatch import MiniBatch
+
+__all__ = ["ItemSampler", "minibatcher_default"]
+
 
-__all__ = ["ItemSampler"]
+def minibatcher_default(batch, names):
```

Comment:
What's `names`?

Reply:
names of item in `batch` such as `seed_nodes`, `node_pairs` which correspond to data attributes of `MiniBatch`.

Reply:
add doc string?

Reply:
added.